html/pfappserver/root/static.alt/src/views/Users/_router/index.js

```diff
@@ -58,13 +55,6 @@ const route = {
       path: 'import',
       name: 'userImport',
       component: UsersImport,
-      props: () => ({ formStoreName: 'formUserImport' }),
```

Comment:
If we remove `formStoreName` here, we need to define it in the component otherwise we through an error.

Reply:
CSV Import script will be refactored separately.